styles/bahmniAppsFormBuilder/_form.scss

```diff
@@ -194,7 +194,6 @@ $lightestGray: #eee;
           &[disabled] {
             background: #eee;
             border: #ccc 1px solid;
-            color: #ccc;
```

Comment:
Can we give a try with `#8d8d8d` and `#c4c4c4` hex codes as well? In that way, we won't end up with pure black text but it will be darker than the #ccc hex colour.

Reply:
I asked about trying different shades of colors. But I was told that this is all we need